tsfile/src/main/java/org/apache/iotdb/tsfile/read/common/Path.java

```diff
@@ -99,19 +96,25 @@ public Path(String device, String measurement) {
     }
     // use PathNodesGenerator to check whether path is legal.
     if (!"".equals(device) && !"".equals(measurement)) {
-      String fullPath = device + TsFileConstant.PATH_SEPARATOR + measurement;
-      PathNodesGenerator.checkPath(fullPath);
+      String path = device + TsFileConstant.PATH_SEPARATOR + measurement;
+      String[] nodes = PathNodesGenerator.splitPathToNodes(path);
+      this.device = transformNodesToString(nodes, nodes.length - 1);
+      this.measurement = nodes[nodes.length - 1];
+      this.fullPath = transformNodesToString(nodes, nodes.length);
     } else if (!"".equals(device)) {
```

Comment:
Maybe we can use `StringUtils.isEmpty()` instead here

Reply:
fixed.